generator/namer.go

```diff
@@ -3,9 +3,14 @@ package generator
 import (
 	"regexp"
 	"strings"
+
+	"github.com/iancoleman/strcase"
 )
 
 const (
+	CaseTitle = "title"
```

Comment:
Defining these as public constants in the `generator` package and importing them in `config` feels bad. Defining them in `config` and importing them here might be slightly better but also seems like a layering violation. Where would you prefer these defined?